workflows/prognostic_run_diags/fv3net/diagnostics/prognostic_run/rundirs.py

```diff
@@ -0,0 +1,223 @@
+"""Utilities for loading computed diagnostics
+
+"""
+import json
+from typing import Iterable, Sequence
+import os
+import numpy as np
+import xarray as xr
+import fsspec
+import pandas as pd
+from pathlib import Path
+from dataclasses import dataclass
+
+
+__all__ = ["ComputedDiagnosticsList"]
+
+
+PUBLIC_GCS_DOMAIN = "https://storage.googleapis.com"
+
+
+@dataclass
+class ComputedDiagnosticsList:
+    """Represents a list of computed diagnostics
+
+    Attrs:
+        url: URL to directory containing rundirs as subdirectories.
```

Comment:
I know this was the preexisting language used, but `rundirs` feels like the wrong name... these are directories with reduced diagnostics/metrics/movies in them, not the actual directories with model run output. Importantly, these paths are different than the ones specified in `rundirs.json`, which has confused me in the past.

Maybe `diagdirs`?

Reply:
Renamed to "computed_diagnostics.py" based on the class name.